core/src/main/java/org/infinispan/interceptors/impl/NotificationInterceptor.java

```diff
@@ -20,7 +19,7 @@
  */
 public class NotificationInterceptor extends DDAsyncInterceptor {
    private CacheNotifier notifier;
-   private final InvocationSuccessHandler transactionCompleteSuccessHandler = new InvocationSuccessHandler() {
+   private final InvocationSuccessAction transactionCompleteSuccessHandler = new InvocationSuccessAction() {
```

Comment:
wouldn't be worth to have 2 ```InvocationSuccessAction```: one for successful transactions and another for unsuccessful transactions?